pom.xml

```diff
@@ -164,9 +164,9 @@
 
   <pluginRepositories>
     <pluginRepository>
-      <id>openmrs-repo</id>
-      <name>OpenMRS Public Repository</name>
-      <url>https://mavenrepo.openmrs.org/public</url>
+      <id>openmrs-repo-releases</id>
+      <name>OpenMRS Releases Repo</name>
+      <url>https://openmrs.jfrog.io/artifactory/releases</url>
```

Comment:
I don't think we need this change; specifically, `public` aliases all the repos in the OpenMRS Maven instance; `releases` does not.

Reply:
This is somewhat connected to [this](https://openmrs.slack.com/archives/C03U2P841/p1682617009756369?thread_ts=1682419613.318009&cid=C03U2P841) suggestion @ibacher.

Reply:
So what's happening is the openmrs packager plugin artefact isn't downloadable via `https://mavenrepo.openmrs.org/public`, but is via `https://openmrs.jfrog.io/artifactory/releases`. Could this be a problem with the alias url or jfrog?

Reply:
I think I get your point @ibacher. In  essence I should use `https://openmrs.jfrog.io/artifactory/public` instead of `https://openmrs.jfrog.io/artifactory/releases`?